server/models/Interests.model.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AllowNull,
+  AutoIncrement,
+  Column,
+  CreatedAt,
+  Length,
+  Model,
+  PrimaryKey,
+  Table,
+  UpdatedAt,
+} from 'sequelize-typescript';
+
+interface InterestsAttributes {
+  interest_id: number;
+  interest_name: string;
+  created_at: Date;
+  updated_at: Date;
+}
+
+@Table({
+  tableName: 'Interests',
+  timestamps: true,
+})
+
+class Interests extends Model<InterestsAttributes> implements InterestsAttributes {
```

Comment:
models should be singular because they describe one thing 
tables should be plural because they house multiple of the same type of things

so for the `Interests` model, change it to `Interest`
same goes for the file name